commands/uploader.go

```diff
@@ -96,17 +107,16 @@ func (c *uploadContext) prepareUpload(unfiltered ...*lfs.WrappedPointer) (*tq.Tr
 		// current committer.
 		var canUpload bool = true
 
-		if lock, ok := c.locks[p.Name]; ok {
-			owned := lock.Committer.Name == c.committerName &&
-				lock.Committer.Email == c.committerEmail
+		if lock, ok := c.theirLocks[p.Name]; ok {
```

Comment:
Nit: I would call this:

```go
if theirs, ok := c.theirLocks[p.Name]; ok {
        // ...
}
```

to be consistent with how we refer to it in the API.